tests/tracing-tests/test-trace-filter.ts

```diff
@@ -81,7 +82,7 @@ describeDevMoonbeam("Trace filter - Contract creation ", (context) => {
     expect(response.result.length).to.equal(1);
     expect(response.result[0].action.creationMethod).to.equal("create");
     expect(response.result[0].action.from).to.equal(ALITH_ADDRESS);
-    expect(response.result[0].action.gas).to.equal("0xb5c2c8");
+    expect(response.result[0].action.gas).to.equal("0x758");
```

Comment:
That is a huge difference, what justifies it ?

Reply:
The old value is near 12M gas (block limit?). So I'd get it corresponds to the target gas of the transaction, which is lowered using estimate_gas.
I wonder why the gas field gets this value tho, it should probably be the gas used instead of the gas limit. (or a reverting contract deployement consume all gas?)

Reply:
@crystalin @nanocryk I confirm that when creating a contract revert, used_gas is gas_limit, whatever the gas_limit provided. I tested by changing gas limit several times.